src/constants/theme.tsx

```diff
@@ -80,6 +85,11 @@ export const themes = {
       checkbox_fill: "#bf40bf", // Radio/checkbox button color
       fill_base: "#222222", // Background color
       border_color_base: "#fffff", // Line color
+      search_bar_fill: "#FFFFFF", // Search Bar fill color
+      radius_sm: 24, // Tag Radius
+      tag_height: 35, // Tag Height
+      button_font_size_sm: 15, // Tag Text Font Size
+      color_text_caption: "#000000", // Tag Text Color
```

Comment:
Is this gonna play well if the background is black? Or are we keeping the tag backgrounds white for dark and light?